scripts/clientd-tests.sh

```diff
@@ -16,20 +16,20 @@ echo $! >> $FM_PID_FILE
 await_server_on_port 8081
 
 #### BEGIN TESTS ####
-[[ $($FM_CLIENTD_CLI info | jq -r 'has("success")') = true ]]
-[[ $($FM_CLIENTD_CLI pending | jq -r 'has("success")') = true ]]
-[[ $($FM_CLIENTD_CLI new-peg-in-address | jq -r 'has("success")') = true ]]
-ADDR=$($FM_CLIENTD_CLI new-peg-in-address | jq -r '.success.peg_in_address');
+[[ $($FM_CLIENTD_CLI info | jq -r 'has("coins")') = true ]]
+[[ $($FM_CLIENTD_CLI pending | jq -r 'has("transactions")') = true ]]
+[[ $($FM_CLIENTD_CLI new-peg-in-address | jq -r 'has("peg_in_address")') = true ]]
+ADDR=$($FM_CLIENTD_CLI new-peg-in-address | jq -r '.peg_in_address');
 
 #for peg-in we need the TxOutProof and a Transaction
 TX_ID="$($FM_BTC_CLIENT sendtoaddress $ADDR 0.001)"
 $FM_BTC_CLIENT generatetoaddress 11 "$($FM_BTC_CLIENT getnewaddress)"
 
 #wait until valid (also test the wait-block-height endpoint)
-[[ $($FM_CLIENTD_CLI wait-block-height  $EXPECTED_BLOCK_HEIGHT | jq -r 'has("success")') = true ]]
-
+[[ $($FM_CLIENTD_CLI wait-block-height  $EXPECTED_BLOCK_HEIGHT | jq -r '.') = "done" ]]
+sleep 15
```

Comment:
Why do we sleep here instead of doing the typical polling?

Reply:
my bad, this was a stupid 'bug' because I initialized the  `EXPECTED_BLOCK_HEIGHT` too early.

Reply:
and I cant use the lib function because it uses the `mint-client-cli` which cant run when `clientd` is active